contrib/zenith_test_utils/zenithtest.c

```diff
@@ -9,17 +9,29 @@
  *-------------------------------------------------------------------------
  */
 #include "postgres.h"
-#include "fmgr.h"
 
 #include "access/xact.h"
+#include "access/relation.h"
+#include "catalog/namespace.h"
+#include "fmgr.h"
+#include "funcapi.h"
+#include "miscadmin.h"
 #include "storage/buf_internals.h"
 #include "storage/bufmgr.h"
+#include "utils/builtins.h"
+#include "utils/rel.h"
+#include "utils/varlena.h"
 
 
 PG_MODULE_MAGIC;
 
 PG_FUNCTION_INFO_V1(test_consume_xids);
 PG_FUNCTION_INFO_V1(clear_buffer_cache);
+PG_FUNCTION_INFO_V1(get_raw_page_at_lsn);
+
+
+extern void zenith_read_at_lsn(SMgrRelation reln, ForkNumber forkNum, BlockNumber blkno,
+			XLogRecPtr request_lsn, bool request_latest, char *buffer);
 
```

Comment:
So, this is an extern declaration for the function that's defined in the 'zenith' module. That's pretty error-prone, it's easy for the declaration to go out of sync with the definition. A warning in a comment in both places noting that dependency would be in order, at the very least.